### DISCUSSION TITLE
API Dostop

### DISCUSSION THREAD

**Discussion by Katzusha** [Katzusha/KerieriResort#1] [General]
trenutno mam API-je kr na neoservu

kosak.andraz@gmail.com
neoserv"sztvkvtcss"2253.

S tem da zdej k sma dva jih bo treba prestavit na git z vs code in na neoservu nastavit da crpa datoteke iz githun-a in jih direktno da na server

razn ce mas mogoce boljso idejo?

**Comment by BernardWasTaken** [comment]
honestly se ne spoznam glih na dostop api-jev prek serverjev tk da sam nared tk ko se teb zdi najbols pa najlazji

**Comment by Katzusha** [comment]
Ja boma pc tkle kr na neoservu mela dokler neo treba prestavit
